arXiv.org.js

```diff
@@ -251,6 +264,17 @@ function parseXML(text) {
 		translate.translate();
 	}
 	else {
+		newItem.publisher = "arXiv";
+		newItem.number = "arXiv:" + articleID;
```

Comment:
This line relies on https://github.com/zotero/zotero/issues/2481#issuecomment-1079754076 being fixed to work for `preprint`